usmqe/api/tendrlapi/user.py

```diff
@@ -0,0 +1,142 @@
+"""
+tendrl REST API.
+"""
+
+import json
+import requests
+import pytest
+from usmqe.api.tendrlapi.common import TendrlApi
+
+LOGGER = pytest.get_logger("tendrlapi.user", module=True)
+
+
+class ApiUser(TendrlApi):
+    """ Main class for interact with REST API - user.
+    """
+    def users(self, asserts_in=None):
+        """ Get users.
+
+        Name:        "GET_users",
+        Method:      "GET",
+        Pattern:     "users",
+
+        Args:
+            asserts_in: assert values for this call and this method
+        """
+        pattern = "users"
+        request = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth)
+        self.print_req_info(request)
+        self.check_response(request, asserts_in)
+        if request.ok:
+            defined_keys = {
+                "email",
+                "username",
+                "name",
+                "role"}
+            for item in request.json(encoding='unicode'):
+                user = item["username"]
+                pytest.check(
+                    item.keys() == defined_keys,
+                    "User {0} should contain: {1}\n\tUser {0} contains: {2}"
+                    .format(user, defined_keys, item.keys))
```

Comment:
Could you reformat this a bit? I don't like the `.format(...)` on the beginning of the line - I'm surprised it's even possible :-). For example, you could save the sting into `msg` variable instead.